docs/source/basics/queries.md

```diff
@@ -128,9 +136,11 @@ const CurrentUserForProfile = gql`
     Login: {{currentUser?.profile}}
   `,
 })
-class ProfileComponent {
+class ProfileComponent implements OnInit, OnDestroy {
+  currentUser: any;
+  private querySubscription: Subscription;
   ngOnInit() {
-    this.data = this.apollo
+    this.querySubscription = this.apollo
```

Comment:
It `QueryRef` and not a `Subscription`. It could but you would have to subscribe to `valueChanges` property (`QueryRef.valueChanges`).

Reply:
Thanks for the feedback.

From what I understand, `watchQuery` returns a `QueryRef`, but this example calls `.watchQuery({...}).valueChanges.subscribe(...)`, which does return a `Subscription`.

Am I misunderstanding?

Reply:
Oh, sure, my bad :)